### ISSUE TITLE
Move away from using GTK?

### ISSUE THREAD

**Issue by mxmilkiib** [taylordotfish/freeze#1]
There is [a problem](http://lists.lv2plug.in/pipermail/devel-lv2plug.in/2016-March/001593.html) in using GTK directly from an LV2 plugin. Would it be possible to migrate to an alternative widget toolkit or framework like DPF? See [here](https://github.com/drobilla/lv2/wiki#gui) for more.

**Comment by taylordotfish** [comment]
Yeah, this is a good point. It's actually why I chose GTK 2 initially, despite having more experience with GTK 3—Ardour uses GTK 2 and can't (or at least couldn't at that time) load GTK 3 plugins exactly for that reason.

Unfortunately I don't have a lot of time to work on this now, but I'd accept a PR that adds an alternative GUI implementation. My advice in the meantime for people having issues due to this problem would be to load the plugin in [Carla](https://github.com/falkTX/Carla), which runs plugin UIs in separate processes.
